tests/test_mg_convert/test_mg_convert.py

```diff
@@ -154,6 +154,11 @@ def _run_openmc(self):
             form = '{0:12.6E} {1:12.6E}\n'
             outstr += form.format(sp.k_combined[0], sp.k_combined[1])
 
+            # Enforce closing statepoint and summary files so HDF5
+            # does not throw an error during the next OpenMC execution
+            sp._f.close()
+            sp._summary._f.close()
```

Comment:
OpenMC throws a whole bunch of errors and quite, all from the HDF5 module.  They all come down to 'cant access file'.  I am on [h5py 2.6.0](https://www.archlinux.org/packages/community/x86_64/python-h5py/) and [hdf5 1.10-patch 1](https://www.archlinux.org/packages/extra/x86_64/hdf5-cpp-fortran/) (I linked to the package files so that if there are build differences that could lead to this, we all can get to the Arch Linux PKGBUILD file).